content/Arduino Cloud/Arduino IoT Cloud/Export-historical-data-in-Arduino-Cloud.md

```diff
@@ -23,7 +23,7 @@ Follow these steps to request a download link:
 
 5. Click the **Get Data** button.
 
-6. An email with the subject **Arduino IoT Cloud historic data** will be sent to the email address associated with your Arduino account. Click the **Download** link inside the email to download the data.
+6. An email with the subject **Arduino Cloud historic data** will be sent to the email address associated with your Arduino account. Click the **Download** link inside the email to download the data.
```

Comment:
Just tested this, and the email still says "Arduino IoT Cloud historic data". I have reported this as a bug, and asked that the email subject is updated to "Arduino Cloud historic data". I think we can just avoid mentioning the subject name here.

```suggestion
6. An email will be sent to the email address associated with your Arduino account. Click the **Download** link inside the email to download the data.
```